<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>机器人控制系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1101</width>
      <height>811</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>主界面</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>680</y>
        <width>271</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>UR Robot 2024</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>121</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>70</y>
        <width>171</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Ry</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit_1">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Rz</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Rx</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_distance">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>580</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>距离</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mm">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>580</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_serOpen">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>640</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_startRaster">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>670</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>开始扫描</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_distance_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>310</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>扫描点的坐标</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>370</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>440</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>510</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>360</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>430</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>500</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>370</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>440</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>510</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>起始X</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_32">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>380</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>起始Y</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_33">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>440</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_81">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>440</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>起始Z</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_52">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>440</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_82">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>380</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>结束Y</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_53">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>380</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_54">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_83">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>结束Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_84">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>320</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>结束X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_85">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>320</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_118">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>380</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_119">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>440</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_120">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>440</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_121">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>320</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_122">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>380</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_123">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>570</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>传感器</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_34">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>500</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_86">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>510</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>列</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_87">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>510</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>行</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_55">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>500</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_distance_3">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>机器人位姿</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_y_dec">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_x_dec">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>X-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_125">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>移动控制</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_x_inc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>X+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_z_dec">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Z-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_y_inc">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Y+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_z_inc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Z+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_startRaster_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>570</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>设置当前点
为起始点</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_startRaster_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>570</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>设置当前点
为结束点</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>点云显示</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>901</width>
        <height>731</height>
       </rect>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>881</width>
         <height>711</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout"/>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>170</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>开启点云</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
